Fix:
sffile.c:38:10: error: implicitly declaring library function 'isprint' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
--- libs/libsffile/sffile.c.orig	2021-02-09 22:01:18.000000000 -0600
+++ libs/libsffile/sffile.c	2021-03-16 02:18:28.000000000 -0500
@@ -20,6 +20,7 @@
  *================================================================*/
 //Interface to Denemo License:  FSF GPL version 3 or later
 
+#include <ctype.h>
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
